frontend/public/extend/devconsole/components/topology/Topology.tsx

```diff
@@ -42,16 +43,25 @@ export default class Topology extends React.Component<TopologyProps, State> {
 
   render() {
     return (
-      <Graph
-        graph={this.props.data.graph}
-        topology={this.props.data.topology}
-        nodeProvider={nodeProvider}
-        edgeProvider={edgeProvider}
-        selected={this.state.selected}
-        onSelect={this.onSelect}
-      >
-        {this.renderToolbar}
-      </Graph>
+      <>
+        <Graph
+          graph={this.props.data.graph}
+          topology={this.props.data.topology}
+          nodeProvider={nodeProvider}
+          edgeProvider={edgeProvider}
+          selected={this.state.selected}
+          onSelect={this.onSelect}
+        >
+          {this.renderToolbar}
+        </Graph>
+        {this.state.selected && (
```

Comment:
Better use a ternary operator here like
```suggestion
        { this.state.selected ? <TopologySidebar /> : null }
```

Reply:
fixed in https://github.com/talamer/console/pull/50/commits/0bfa567f5acfde6f42c51e2301f17f5358aa4f2b